crates/meroctl/src/main.rs

```diff
@@ -1,13 +1,17 @@
 use std::process::ExitCode;
 
 use clap::Parser;
+use rand::Rng;
+use reqwest::Client;
 
 use crate::cli::RootCommand;
+use crate::version::check_for_update;
```

Comment:
these should actually be on line 15, since they're symbols from submodules

```suggestion
use cli::RootCommand;
use version::check_for_update;
```

Reply:
I thought cargo +nightly fmt fixes that

Reply:
okayy